tests/lib/kubernetes/caasp.py

```diff
@@ -69,25 +67,28 @@ def _caasp_init(self):
                 ['skuba', 'cluster', 'init', '--control-plane',
                  self.hardware.masters[0].get_ssh_ip(),
                  self._clusterpath],
-                env=env, check=True)
+                env=env, check=True, capture_output=True)
```

Comment:
Won't this stop any useful output to the console unless there is a failure?
It's probably not necessary, but at least makes it obvious the test is doing something.

We should instead redirect the output to logging (this goes for other places too).

Reply:
Any suggestion how to do that? Then problem I'm trying to solve is, that we don't have stdout/stderr in an error case so it's hard to find out what was wrong

Reply:
Also, if everything goes well, I'm not interested in the output :)